pkgs/development/libraries/kquickimageedit/0.3.0.nix

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
You should apply the same set of changes you have for pkgs/development/libraries/kquickimageedit/default.nix (finalAttrs etc) to this file.